ocs_ci/templates/workloads/pgsql/PGSQL_Benchmark.yaml

```diff
@@ -11,11 +11,11 @@ spec:
       clients:
         - 20
       threads: 2
-      transactions: 100
+      transactions: 15000
```

Comment:
Any reason for changing the default parms?
This change effects all the pgsql tests. With these changes pgsql runs for a longer duration which can increase the workloads run time. For functional workload tests the current values holds good.